apps/coursition/src/app/actions.ts

```diff
@@ -34,19 +34,20 @@ export const createUser = async ({ email, password }: { email: string; password:
         paymentStatus: 'FREE',
       },
     })
-    .catch((err: unknown) => {
-      if (err instanceof PrismaClientKnownRequestError) {
-        if (err.code === 'P2002') {
+    .catch((error: unknown) => {
+      if (error instanceof PrismaClientKnownRequestError) {
+        if (error.code === 'P2002') {
           return { error: 'An account already exist with this email' }
```

Comment:
i  see now that we should always use `error` as suppoesed to `err`, is that a convention that you're trying to implement?

Reply:
Yes, it is recommended via OxLint. :)